torch/csrc/inductor/aoti_torch/c/shim.h

```diff
@@ -677,6 +677,15 @@ aoti_torch_library_def(TorchLibraryHandle self, const char* schema);
 AOTI_TORCH_EXPORT AOTITorchError
 aoti_torch_delete_library_object(TorchLibraryHandle tlh);
 
+// calls the op overload defined by a given opName, overloadName, and a
+// stack of StableIValues. This call will populate any return values of the
+// op into the stack in their StableIValue form, with ret0 at index 0, ret1
+// at index 1, and so on.
```

Comment:
todo after this: we should document our StableIValue stack definition and expose to/from helpers for it.

Reply:
I actually did that, just in library.h https://github.com/pytorch/pytorch/blob/main/torch/csrc/stable/library.h#L69